### PR TITLE
Add views and url to VideoInfo

### PR DESCRIPTION
**Pull request by prateekmedia**
cc @FireMasterK, I hope this will fix TeamPiped/piped_dart#6

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -376,6 +376,12 @@ components:
         uploaderVerified:
           type: boolean
           description: Whether the video's uploader is verified.
+        url:
+          type: string
+          description: The relative URL to the video.
```
Comment (FireMasterK) [inline swagger.yaml]: Are you sure this exists here? I don't think it's included in this response as you require the video id to make the request

```
curl -s -o - https://pipedapi.kavin.rocks/streams/gsMIT25iPXk | jq -r .url
```

Comment (prateekmedia) [reply swagger.yaml]: yes, but many actions in the UI like sharing video depends upon the url, how will it work out?

Comment (FireMasterK) [reply swagger.yaml]: That would depend on your application's implementation, this isn't currently available in the API.

There's a lot of URL formats, so I don't think it's a good idea to force clients to choose:
```
https://www.youtube.com/watch?v={videoId}
```
```
https://piped.kavin.rocks/watch?v={videoId}
```
```
https://youtu.be/{videoId}
```

Etc.

Comment (prateekmedia) [reply swagger.yaml]: Ok understood, you can squash and merge this if you think everything is ok

**Pull request merged**